.gitignore

```
@@ -61,4 +61,6 @@ typings/
 .next
 
 # package.lock.json
-package-lock.json
\ No newline at end of file
+package-lock.json
+/build/1.e8080218.js
+/build/index.html
```

```suggestion
build/
```
No need to specify every file.